Fixed Indexer fails with "Object is missing required member 'id'" (#2568)
magda-io/magda: CHANGES.md

-   Fixed indexer incorrect parsing bounding box data in spatialCoverage aspect
-   Added auth when crawling the registry
-   Fixed Data.json spatial bounding box ordering not understood

> -   Indexer fails with "Object is missing required member 'id'"

Reviewer: Maybe word this change as:
```suggestion
-   Handle and log the exception "Object is missing required member 'id'"
```

Author: Thanks~ That reads better~ Will update~ 👍
